test/test_package.py

```diff
@@ -261,6 +261,18 @@ def test_validate_package(self):
             self.assertRaises(InvalidPackage, Package.validate, pack)
             dep_type.remove(depend)
 
+    def test_invalid_package_exception(self):
+        try:
+            raise InvalidPackage('foo')
+        except InvalidPackage as e:
+            self.assertEqual('foo', str(e))
+            self.assertEqual(None, e.package_path)
+        try:
+            raise InvalidPackage('foo', package_path='\\bar')
+        except InvalidPackage as e:
+            self.assertEqual('Error(s) in package \'\\bar\':\nfoo', str(e))
```

Comment:
Same here regarding escaping the quotes.

Reply:
Fixed.

Reply:
BTW maybe there is a way to have a flake8 check for cases like this, so human spend time on more important stuff.

Reply:
In ROS 2 we have these kind of checks in place. Not for long existing ROS 1 packages though at the moment...

Reply:
I have PR #211  that at some level addresses this issue.  Happy to bring other checks from ament_flake8 later.  Don't want to do it all in one change - fixing a lot of flake8 issues is hard to review.